### ISSUE TITLE
Small improvement proposal on Legend class

### ISSUE THREAD

**Issue by dmajere** [weaveworks/grafanalib#26]
There are four class fields that define what type of values grafana will show in legend

```
    avg = attr.ib(default=False, validator=instance_of(bool))
    current = attr.ib(default=False, validator=instance_of(bool))
    max = attr.ib(default=False, validator=instance_of(bool))
    min = attr.ib(default=False, validator=instance_of(bool))
```

But, grafana won't actually show them if you wont set `values` field to True too.
May be we should automatically set `values` to True if any of `avg`, `current`, `max` or `min` are set to True

**Comment by jml** [comment]
SGTM. `values` should default to `None`, and if it is `None`, we'll set its value to `any([avg, current, max, min])`. If it's set explicitly, we should respect that explicit setting.

**Comment by dmajere** [comment]
Closed by #30
